misc/python/materialize/mzbuild.py

```diff
@@ -176,6 +176,11 @@ def inputs(self) -> Set[str]:
             # *lot* of work.
             "Cargo.lock",
             ".cargo/config",
+            # Crate roots can contain mzcompose files, but mzcompose should
+            # never impact the Rust build process, so exclude them from the
```

Comment:
The only situation where they could conduct the build process is if we build docker images intentionally with mzcompose files inside of them, but at that point we will need to think harder about things. In particular, what we're doing with our lives.

Reply:
Lol yep, my thoughts exactly. (You could also `include!` them into a Rust binary, but your point applies double in that case.)